internal/testutil/fs.go

```diff
@@ -27,7 +28,7 @@ func TestTempDir(tb testing.TB) string {
 // TestOpenFile creates a new file called name and returns the opened file.
 func TestOpenFile(tb testing.TB, name string) *os.File {
 	tb.Helper()
-	f, err := os.OpenFile(name, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0600)
+	f, err := os.OpenFile(filepath.Clean(name), os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0600)
```

Comment:
This is test-only code so I don't think gosec is making a good recommendation here, but let's leave it.